gensim/models/_fasttext_bin.py

```diff
@@ -366,5 +386,166 @@ def _backslashreplace_backport(ex):
     return text, end
 
 
+def _sign_model(fout):
+    # Reimplementation of the
+    # [FastText::signModel](https://github.com/facebookresearch/fastText/blob/master/src/fasttext.cc)
+    fout.write(_FASTTEXT_FILEFORMAT_MAGIC.tobytes())
+    fout.write(_FASTTEXT_VERSION.tobytes())
+
+
+def _conv_field_to_bytes(field_value, field_type):
+    if field_type == 'i':
+        return (np.int32(field_value).tobytes())
+    elif field_type == 'd':
+        return (np.float64(field_value).tobytes())
+    else:
+        raise NotImplementedError('Currently conversion to "%s" type is not implemmented.' % field_type)
+
+
+def _get_field_from_model(model, field):
+
```

Comment:
Dtto: docstrings instead of the blank lines please.

Reply:
Corrected (also in the next functions).